.github/workflows/translations.yml

```diff
@@ -1,66 +1,87 @@
-name: Update translation source
+name: Update Translation Source
 
 on:
   workflow_dispatch:
   schedule:
-    # runs once per month
+    # runs at the start of each month (UTC)
     - cron: '0 0 1 * *'
+  pull_request:
```

Comment:
I don't see a need to do this via PRs, as cron is more than enough. I'd increase the frequency to every 2 weeks however

Reply:
It won't create a PR when the run is triggered from a PR (like this one).
The purpose is only to verify and check changes made to the workflow itself which would not be possible otherwise - that's why it's limited to only run on changes to that one config file.
I should add the related bash script there as well actually.

Reply:
the run triggered by pr would be useless, if you're not testing the creation of the translation pr (preferably on a fork) what's the point?

Reply:
The PR creation is only one step of the workflow.
The logs give useful information in many other areas, e.g. if switching from Qt 5 to 6 for lupdate would break something...
Isn't it just good practice to always check and verify changes that are made by people on the internet?

This 30s action running in a PR happens probably like every 100th PR that gets opened... 🙄

Reply:
of course it is good to verify changes, and you should do so in a proper fashion, a dry run like that will point out simple typos in the workflow but you should be testing the workflow with an automatically generated pr anyway, so I don't see a benefit to having this trigger like this because well, as you mentioned this will rarely happen.

Reply:
There is use in having it (also no harm). Even if you see only little benefit.

I would much rather cut down on our huge general build matrix. Bionic accounts for 0,2% of all 2.8.0 downloads... 🙈

Reply:
it doesn't matter how many people download bionic, we need to verify our minimum requirements with that build

Reply:
My point was a different one, even though the example was maybe a bad pick.

Anyway, not sure if the topic is basically worth staling a PR for one year.